1906-12-10.xml

```diff
@@ -1530,491 +1530,4988 @@
+                            lower than at the retail shops in Cairo and Alexandria and varying from
+                            13 francs to 150 francs the square metre. Besides a grand selection of
+                            modern Oriental carpets of any size and colour Messrs. Sparta'i are
+                            prepared to show a choice collection of fine antique rugs and silk rugs
+                            of the very finest quality and at moderate prices. They are likewise
+                            prepared to execute orders for clients for carpets of any size and
+                            colour desired. We would mention that their goods are most suitable for
+                            Christmas and New Year's presents to relatives and friends in Europe and
+                            America. The firm warm’y invite residents and travelllers to favour
+                            their branches in Egypt with a call to inspect the stock just arrived
+                            from Smyrns and new being exhibited, with the assurance that visitors
+                            will not be importuned to purchase goods they may not require.</p>
+                    </div>
+                    <div type="item">
+                        <p>Captain Hollefen, oommandant of the Ger-man training ship Storch, who was
+                            granted audience by the Khedive at Abdeen Palace on Friday, returned to
+                            Alexandria on Saturday, from Cairo. The Stosch sails on Saturday next.
+                        </p>
+                    </div>
+                    <div type="item">
+                        <p>The Rev P. B. N. Neman-Lee, C P., is expected to arrive by the troopship
+                            Braemar Castle on Thursday next, and will forthwith resume his duties as
+                            Chaplain to the Forces in Alexandria, which since his departure in July
+                            last have been undertaken by tbe clergy of 8t. Mark’s Church. Canon Ward
+                            ceased to be Acting Chaplain to the Forces on December 1.</p>
+                    </div>
+                    <div type="section" element="local">
+                        <head>PERSONAL AND SOCIAL</head>
+                        <p>On their visit to Egypt this month the Earl and Countess of Halsbury will
+                            be accompanied by Mr. and Lady Evelyn Giffard.</p>
+                    </div>
+                    <div type="item">
+                        <head>SERIOUS TRAIN COLLISION.</head>
+                        <p>TWENTY-ONE PASSENGERS INJURED.</p>
+                        <p>The following telegram was received by us on Saturday evening The Railway
+                            Administration regrets to report a serious collision which occurred at
+                            Mit Assas this morning and in
```

Comment:
The cheap prepaid advertising section should close here, with all of the ads above inside of it.